Neos.Neos/Resources/Private/Fusion/Prototypes/Shortcut.fusion

```diff
@@ -1,15 +1,133 @@
+        condition = true
+        renderer = Neos.Fusion:Renderer {
+          fallback = 'no target has been selected'
+          renderer = ${props.i18n.id('shortcut.noTargetSelected').value(this.fallback).translate()}
+        }
+      }
+    }
+    @glue = "<br/>"
+  }
+
+  renderer.selectedTarget.ifTarget.condition {
+    condition = ${props.target}
+    renderer = Neos.Fusion:Match {
+      @subject = ${props.targetSchema}
+
+      node = Neos.Fusion:Renderer {
+        targetUriTag = Neos.Neos:NodeLink {
+          node = ${props.targetConverted}
+        }
+        fallback = afx`
+          Click {this.targetUriTag} to continue to the page.
+        `
+        renderer = ${props.i18n.id('shortcut.clickToContinueToPage').arguments([this.targetUriTag]).value(this.fallback).translate()}
+      }
+
+      asset = Neos.Fusion:Renderer {
+        targetUriTag = Neos.Fusion:Tag {
+          tagName = 'a'
+          attributes {
+            href = Neos.Fusion:ResourceUri {
+              resource = ${props.targetConverted.resource}
+            }
+            target = "_blank"
+          }
+          content = ${props.targetConverted.label}
+        }
+        fallback = afx`
+          Click {this.targetUriTag} to see the file.
+        `
+        renderer = ${props.i18n.id('shortcut.clickToContinueToAsset').arguments([this.targetUriTag]).value(this.fallback).translate()}
+      }
+
+      @default = Neos.Fusion:Renderer {
+        targetUriTag = Neos.Fusion:Tag {
+          tagName = 'a'
+          attributes {
+            href = ${props.target}
+            target = "_blank"
+          }
+          content = ${props.target}
+        }
+        fallback = afx`
+          Click {this.targetUriTag} to open the link.
+        `
+        renderer = ${props.i18n.id('shortcut.clickToContinueToExternalUrl').arguments([this.targetUriTag]).value(this.fallback).translate()}
+      }
+    }
+  }
+
+  // targetMode == 'firstChildNode'
+  renderer.firstChildNode = Neos.Fusion:Renderer {
+    targetUriTag = Neos.Neos:NodeLink {
+      node = ${props.firstChildNode}
+    }
+    fallback = afx`
+      This is a shortcut to the first child page.<br/>
+      Click {this.targetUriTag} to continue to the page.
+    `
+    renderer = ${props.i18n.id('shortcut.clickToContinueToFirstChildNode').arguments([this.targetUriTag]).value(this.fallback).translate()}
+  }
+
+  // targetMode == 'parentNode'
+  renderer.parentNode = Neos.Fusion:Renderer {
+    targetUriTag = Neos.Neos:NodeLink {
+      node = ${props.node.parent}
+    }
+    fallback = afx`
+      This is a shortcut to the parent page.<br/>
+      Click {this.targetUriTag} to continue to the page.
+    `
+    renderer = ${props.i18n.id('shortcut.clickToContinueToParentNode').arguments([this.targetUriTag]).value(this.fallback).translate()}
+  }
+
+  renderer.@process.contentWrap = afx`
+    <div id="neos-shortcut">
+      <p>
+        {value}
+      </p>
+    </div>
+  `
+
+  @cache {
```

Comment:
Why is the caching configuration needed when it was not once it was a template?

Reply:
i just found it out while testing, and it annoyed me:
when you have visited a Shortcut in the backend and then change the user language, the cache will still show you the previous translation in the backend.

... i now since this is hardly ever the case i could also remove it right?